From 1b5ebfad33d00fcca04b56fe9d0e0d1f1e6c223e Mon Sep 17 00:00:00 2001
From: Bjorn Lu <bjornlu.dev@gmail.com>
Date: Fri, 13 Mar 2026 18:10:20 +0800
Subject: [PATCH] docs(releases): update v7 latest minor (#21842)

---
 docs/.vitepress/theme/components/SupportedVersions.vue | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/.vitepress/theme/components/SupportedVersions.vue b/docs/.vitepress/theme/components/SupportedVersions.vue
index b2530ee44febcf..51ac64ee97695c 100644
--- a/docs/.vitepress/theme/components/SupportedVersions.vue
+++ b/docs/.vitepress/theme/components/SupportedVersions.vue
@@ -18,6 +18,7 @@ const previousMajorLatestMinors: Record<string, string> = {
   '4': '4.5',
   '5': '5.4',
   '6': '6.4',
+  '7': '7.3',
 }
 
 // Current latest Vite version and support info
